torch/fx/experimental/symbolic_shapes.py

```diff
@@ -156,7 +156,7 @@ def wrap_int(self, num):
 
     def wrap_float(self, num):
         assert isinstance(num, float)
-        return SymNode(sympy.Integer(num), self.shape_env, float, constant=num)
+        return SymNode(sympy.Float(num), self.shape_env, float, constant=num)
```

Comment:
This is the ONLY logic change left in this PR. All other issues are skipped in the test